### ISSUE TITLE
Why requirements are PHP > 7.1 and not 7.0 ?

### ISSUE THREAD

**Issue by hugsbrugs** [umpirsky/country-list#77]
Hey I have a regular Ubuntu stable release and I can't update your package because of PHP requirements. Is there a reason why 7.1 and not 7.0 ?
Thanks

**Comment by umpirsky** [comment]
Not specifically. I have Ubuntu 17.10 and my version is `PHP 7.1.11-0ubuntu0.17.10.1`. You are probably runing LTS.

**Comment by hugsbrugs** [comment]
Would it be possible to revert to 7.0 not to have to manually install latest PHP version and because of compatibility over different servers config ? Thanks !

**Comment by umpirsky** [comment]
Done.

**Comment by hugsbrugs** [comment]
You're a killer !

**Comment by umpirsky** [comment]
Just checked on old Ubuntu:

```
php -v
PHP 7.1.14-1+ubuntu16.04.1+deb.sury.org+1 (cli) (built: Feb  9 2018 09:33:27) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2018 Zend Technologies
   with Zend OPcache v7.1.14-1+ubuntu16.04.1+deb.sury.org+1, Copyright (c) 1999-2018, by Zend Technologies

cat /etc/issue
Ubuntu 16.04.3 LTS \n \l
```

**Comment by Amadox** [comment]
could you release a new version with this fix please? 2.0.3 still has 7.1 in it.

**Comment by umpirsky** [comment]
@Amadox Released `2.0.4`. ;)
